### PR TITLE
Expose memory.use_hierarchy in MemoryStats

### PR DESCRIPTION
**Pull request by derekwaynecarr**
Fixes https://github.com/opencontainers/runc/issues/1376

Signed-off-by: Derek Carr <decarr@redhat.com>

### CODE REVIEW

**Comment by mrunalp** [comment]
@vbatts TRAVIS_COMMIT_RANGE isn't getting set for this PR failing git-validation. What was the workaround for that?

**Comment by mrunalp** [comment]
We could do something like this https://github.com/mendersoftware/mendertesting/pull/9/files

**Comment by vbatts** [comment]
@mrunalp let me check on that right now

**Comment by mrunalp** [comment]
#1382 should fix the issue with travis.

**Comment by mrunalp** [comment]
LGTM

File: libcontainer/cgroups/fs/memory.go [hunk 1]
```diff
@@ -243,6 +243,14 @@ func (s *MemoryGroup) GetStats(path string, stats *cgroups.Stats) error {
 	}
 	stats.MemoryStats.KernelTCPUsage = kernelTCPUsage
 
+	use_hierarchy := strings.Join([]string{"memory", "use_hierarchy"}, ".")
```
Comment (runcom) [inline libcontainer/cgroups/fs/memory.go]: nit: should be `UseHierarchy`

**Comment by derekwaynecarr** [comment]
@runcom @mrunalp - fixed nit.

**Comment by mrunalp** [comment]
LGTM

**Comment by runcom** [comment]
:+1:

**Comment by derekwaynecarr** [comment]
@mrunalp @vishh -- anything else needed on this?  i wanted to get back around to fixing cAdvisor

**Comment by dqminh** [comment]
LGTM

**Pull request merged**